### PR TITLE
Rework collectors doc

### PR DESCRIPTION
**Pull request by trasher**

### CODE REVIEW

File: source/modules/configuration/index.rst [hunk 1]
```diff
@@ -1,13 +1,13 @@
 Configuration
 =============
 
-Le module Configuration permet d'accéder aux options de configuration générale de GLPI : la configuration globale de GLPI, les notifications, les collecteurs, les tâches automatiques, l'authentification, les plugins, les critères d'unicité et les liens externes protocolés.
+Configuration module gives access to all GLPI configuration options: global configuration, notifications, collectors,  automated tasks,authentication, plugins, unicity criteria ans external links.
 
 Les intitulés  ne proposent pas d'actions spécifiques, se reporter aux :doc:`actions communes <../generalites/actions>`.
```
Comment (trasher) [inline source/modules/configuration/index.rst:6]: I do not really understand this line; I wonder if it should be dropped

File: source/modules/configuration/collectors.rst [hunk 2]
```diff
@@ -1,6 +1,39 @@
...
+A mail collector is associated with an email address. It is possible to add as many collectors as you want. Of course, the more collectors you configure, the more import will take time.
+
+To declare a new collector, you will have to provide a server name, as well as connection options (IMAP or POP, SSL, TLS, certificate validation). It is possible to limit the maximum size for attachments (no import, or from 1 to 100 Mio). This configuration value is inherited from the main configuration (`Setup -> General -> Assistance`)
+A configuration option permit to switch tickets creation date between import date or email date.
+
+Optionally, you can set an archive directory in the mailbox where imported and refused emails will be stored.
+
+Collectors that have errors will be placed at the top of the list. In that case, number of successive errors will be displayed in collector configuration form.
+
+Important: collector name must be the email address used. This allows to determine the address associated to the account. This is used to prevent collector address to be added as observer in tickets.
```
Comment (cedric-anne) [inline source/modules/configuration/collectors.rst:17]: 1. Using a valid email address as collector name is not mandatory.
2. When collector mail is an address, emails that are sent by this address are not collected from this collector.

See https://github.com/glpi-project/glpi/pull/8853.

File: source/modules/configuration/collectors.rst [hunk 3]
```diff
@@ -1,6 +1,39 @@
...
+
+Once the form validated, you can test connection and messages retrieving using the dedicated button.
+
+An automatic action will do the import on configured collectors; time between two executions can be configured in automated task form.
+
+Another automatic action will send notifications when any collector repetitively fails to import emails. See `notifications configuration <modules/configuration/notifications>`.
```
Comment (cedric-anne) [inline source/modules/configuration/collectors.rst:23]: Are you sure this feature exists ?

File: source/modules/configuration/collectors.rst [hunk 4]
```diff
@@ -1,6 +1,39 @@
...
+A mail collector allow to import emails from a mailbox ans turn them into GLPI tickets. A :ref:`routing mechanism <collectors_rules>` will bring them to the destination entity.
+
+A mail collector is associated with an email address. It is possible to add as many collectors as you want. Of course, the more collectors you configure, the more import will take time.
+
+To declare a new collector, you will have to provide a server name, as well as connection options (IMAP or POP, SSL, TLS, certificate validation). It is possible to limit the maximum size for attachments (no import, or from 1 to 100 Mio). This configuration value is inherited from the main configuration (`Setup -> General -> Assistance`)
+A configuration option permit to switch tickets creation date between import date or email date.
+
+Optionally, you can set an archive directory in the mailbox where imported and refused emails will be stored.
+
+Collectors that have errors will be placed at the top of the list. In that case, number of successive errors will be displayed in collector configuration form.
```
Comment (orthagh) [inline source/modules/configuration/collectors.rst:15]: The previous phrase lets thinking there is an order about errors count in the list.
This is not true, collector in errors are displayed normally in the list (default order by name) but a warning above the list show collectors in errors

```suggestion
Collectors that have errors will be indicated above the list. In that case, number of successive errors will be displayed in collector configuration form.
```

File: source/modules/configuration/collectors.rst [hunk 5]
```diff
@@ -1,6 +1,39 @@
...
+
+Important: collector name must be the email address used. This allows to determine the address associated to the account. This is used to prevent collector address to be added as observer in tickets.
+
+Once the form validated, you can test connection and messages retrieving using the dedicated button.
```
Comment (orthagh) [inline source/modules/configuration/collectors.rst:19]:
```suggestion
Once the form validated, you can test connection and messages retrieving by using the dedicated button.
```

File: source/modules/configuration/index.rst (hunk 1 above)
Comment (orthagh) [reply source/modules/configuration/index.rst:6]: it's an introduction to the menu and what you have access in it.
Not super usefull, nor useless

File: source/modules/configuration/collectors.rst (hunk 3 above)
Comment (trasher) [reply source/modules/configuration/collectors.rst:23]: Not at all; I've just translated :D

File: source/modules/configuration/collectors.rst (hunk 2 above)
Comment (trasher) [inline source/modules/configuration/collectors.rst:17]:
```suggestion
```

File: source/modules/configuration/index.rst (hunk 1 above)
Comment (trasher) [reply source/modules/configuration/index.rst:6]: Surely misplaced then... Well, I won't change that; goal was to rework collectors only

File: source/modules/configuration/collectors.rst (hunk 3 above)
Comment (trasher) [reply source/modules/configuration/collectors.rst:21]: It seems to be the goal of the `mailgateerror` automatic action

**Review by cedric-anne — APPROVED** [review]

**Review by orthagh — APPROVED** [review]

**Pull request merged**